lib/bitcore.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
```

Comment:
shouldn't this file be called `chains.js` instead of `bitcore.js` ?

Reply:
This isn't the adapter idea yet. This is just having one file that can configure the bitcore library rather than requiring it directly.